src/plugins/table/table-utils.spec.js

```diff
@@ -1,5 +1,3 @@
-/* eslint-disable max-lines */
```

Comment:
Good. I have always disliked this rule. The decision to split a file should be based on logic not on a counter 😬